### ISSUE TITLE
Release not appearing in list.

### ISSUE THREAD

**Issue by probityrules** [SpringRoll/SpringRollConnect#2]
I added a new release and received a confirmation that it was successfully created, but it never appeared on the list of releases (even with a page refresh). However, it appears to exist, since I am able to pull the built release via the test link and if I try to re-add it, I receive a notification that the commit id is already taken.

![untitled](https://cloud.githubusercontent.com/assets/1324745/10495498/302a899c-728a-11e5-8a6a-14546d6acfc9.png)

**Comment by bigtimebuddy** [comment]
Not sure what happened here. @brianmlink made a whitespace commit which added a new release and that was successful. We are looking at some possible issue. Let me know if this happens again. I'm going to leave this open for now.

**Comment by probityrules** [comment]
I just had it happen again with the same game. I'm fairly certain this time was a different sequence of events leading to the issue, but hopefully it will shed a bit more light on things:

1. I pushed a build to Stash. A few minutes later it showed up in the list on SR Connect automatically. Everything worked as expected.
2. I pushed another build to Stash a few minutes later. This time it didn't show up on SR Connect, even after waiting 10-15 minutes.
3. I manually added the latest commit by copying the commit id from Stash into the +Add dialog on SR Connect. The newly added build appeared in the list.
4. However, when I tried to pull the game for testing (latest dev build), I received a 404 and this message: NoSuchKeyThe specified key does not exist.river-run/0831d57da5f3fea073a0c7ec9dcb7239665a7681/debug/index.htmlCB8B60580AA12C66GBsaJQXVcBFVEj3k4kqJJAGeRoXKj11hcwo2JT6KvVpsZKWf/qpIp2LPS1VLpvkh
5. Realizing something was corrupted, I deleted the current instance from the list. The latest dev link (from ernie) now works (pulling the earlier version).
6. I then re-added the latest commit (like 3 above).
6. This time it was successfully added, but no longer shows in the list. Trying to pull the latest dev build results in the same 404 error.

I can leave it as is if it helps anyone. Otherwise I'll try making a negligible commit to see if it resolves itself like last time.
